front/app/components/admin/ContentBuilder/Widgets/Iframe/utils.ts

```diff
@@ -44,8 +11,5 @@ export const isValidUrl = (url: string) => {
   if (!validUrlRegex.test(url)) {
     return [false, 'invalidUrl'];
   }
-
-  const isAcceptableUrl = urlWhiteList.some((rx) => rx.test(url));
-
-  return [isAcceptableUrl, 'whitelist'];
+  return [true, 'validUrl'];
```

Comment:
Minor: We can also now simplify this logic and just return a boolean. The usage can set 'invalidUrl' as the boolean would make it direct to understand.